dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -178,7 +178,7 @@ export default defineComponent({
           ? kpiValue.map((naceCodeShort: string) => naceCodeMap.get(naceCodeShort)?.label ?? naceCodeShort)
           : naceCodeMap.get(kpiValue as string)?.label ?? kpiValue;
       }
-      if (kpiField.name.includes("Countries")) {
+      if (kpiField.name.includes("Countries") && kpiField.component !== "YesNoFormField") {
         kpiValue = Array.isArray(kpiValue)
           ? kpiValue.map(
               (countryCodeShort: string) => getCountryNameFromCountryCode(countryCodeShort) ?? countryCodeShort
```

Comment:
The expression behind the `return` statement is underlined red in the IDE for me.
Do you observe this too?
If yes, do you have an idea why?

Reply:
It was a bit picky in IntelliJ. Perhaps even a false positive. The fact that it compiles would support that fcat. No warning/error in VS COde.
I changed filter()[0] to .find() because it does the same thing.
Done.

Reply:
Interesting: Your change to `.find()` fixed the red underlining in my IDE.

👍

Reply:
Actually it was the additional `if(kpiField.options?.length)` that fixed it. No matter what I did in the filter() or the find(), it did not want to remove the red. Even the most basic of statements would show red line.